strikecircle/tasks.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+from .env import env
+from logging import getLogger
+from django.core.serializers import serialize
+from strikecircle.models import Pledge, StrikeCircle
+
+from airtable import Airtable
+from background_task import background
+
+
+logger = getLogger(__name__)
+
+
+API_KEY = env("AIRTABLE_API_KEY")
+BASE_KEY = env("AIRTABLE_BASE_KEY")
+
+
+# TODO: Deploy supervisord in prod to run tasks https://django-background-tasks.readthedocs.io/en/latest/#running-tasks
+
+
+@background(schedule=10, queue="airtable-export")
+def export_strike_circle_to_airtable(strike_circle_id):
+	"""
+	type: (int) -> Dict[str, Any]
+	Export Strike Circle to Airtable so administraters can analyze data.
+	"""
+	try:
+		# Passing raw_record as list because serialization only works for iterable objects: https://code.djangoproject.com/ticket/11244
+		strike_circle = StrikeCircle.objects.get(pk=strike_circle_id)
+		# TODO: Consider alternatives to serialization for filtering https://stackoverflow.com/a/29088221/4710041
```

Comment:
I read that SO answer -- I'm good with serialization for filtering/converting the model object to a dict for now. I think a comment mentioning that part of the point of serialization is to turn the model record into a dict, because that wasn't initially clear to me.